mappings/net/minecraft/item/tool/ItemHoe.mapping

```diff
@@ -1,2 +1,19 @@
 CLASS none/aen net/minecraft/item/tool/ItemHoe
+	FIELD a material Lnone/aeo$a;
+	FIELD b swingSpeed F
+	METHOD <init> (Lnone/aeo$a;)V
+		ARG 0 material
 	METHOD a activate (Lnone/aak;Lnone/aiu;Lnone/cn;Lnone/qz;Lnone/cu;FFF)Lnone/rb;
+	METHOD a tillSoil (Lnone/aeq;Lnone/aak;Lnone/aiu;Lnone/cn;Lnone/asj;)V
```

Comment:
Yes, this method is only used by the hoe, and is only in the hoe class. It's basically a fancy way for the how to set a block and play it's sound.